test/unit/pr_checker.test.js

```diff
@@ -800,4 +806,74 @@ describe('PRChecker', () => {
       cli.assertCalledWith(expectedLogs);
     });
   });
+
+  describe('checkPRStatus - check PR status (closed || merged)', () => {
+    let cli = new TestCLI();
+
+    afterEach(() => {
+      cli.clearCalls();
+    });
+
+    it('should warn if PR is closed', () => {
+      const expectedLogs = {
+        warn: [
+          [ 'This PR is closed' ]
+        ]
+      };
+
+      const options = {
+        pr: closedPR,
+        reviewers: allGreenReviewers,
+        comments: [],
+        reviews: [],
+        commits: simpleCommits,
+        collaborators
+      };
+
+      const checker = new PRChecker(cli, options, argv);
+      const status = checker.checkPRState();
+      assert.strictEqual(status, false);
+      cli.assertCalledWith(expectedLogs);
+    });
+
+    it('should warn if PR is merged', () => {
+      const expectedLogs = {
+        warn: [
+          [ 'This PR is merged' ]
```

Comment:
We use `[abc]` instead of `[ abc ]` (no spaces around brackets) in node.js core. Personally, I am not a fan of that convention, just note that this differs.

Reply:
Ah.., we do not have any lint rules for this and so we do have mixed up array with and without spaces.

Reply:
I think the current rule is: as long as you are being consistent, i.e. not something like `[ 'something']`, you are fine.